Formula/cpplint.rb

```diff
@@ -0,0 +1,24 @@
+class Cpplint < Formula
+  include Language::Python::Virtualenv
+
+  desc "Static code checker for C++"
+  homepage "https://github.com/cpplint/cpplint"
+  url "https://github.com/cpplint/cpplint/archive/1.5.4.tar.gz"
+  sha256 "e254b5620fb039689b8f5e64b07384ee53beb695304c5c01195133be662b4457"
+  license "Apache-2.0"
+
+  livecheck do
+    url :stable
+  end
+
+  depends_on "python@3.8"
+
+  def install
+    virtualenv_install_with_resources
```

Comment:
There are no resources defined, did you forget them?

Reply:
It doesn't need resources, like youtube-dl.

https://github.com/Homebrew/homebrew-core/blob/bd657790c9fc73913988720a6d9c5ecf3070bf70/Formula/youtube-dl.rb#L1-L29